crates/wash/src/cli/call.rs

```diff
@@ -182,34 +182,38 @@ pub struct ConnectionOpts {
     /// RPC Host for connection, defaults to 127.0.0.1 for local nats
     #[clap(
         short = 'r',
-        long = "rpc-host",
-        env = "WASMCLOUD_RPC_HOST",
+        long = "nats-host",
+        alias = "rpc_host",
```

Comment:
```suggestion
        alias = "rpc-host",
```

You were on the right track here! Turns out `clap` will automatically convert snake case into kebab case for the CLI, so the aliases should be kebab case too!

```bash
➜ wash call --help|grep -A1 rpc-host
  -r, --rpc-host <RPC_HOST>
          RPC Host for connection, defaults to 127.0.0.1 for local nats [env: WASMCLOUD_RPC_HOST=] [default: 127.0.0.1]
```

Reply:
Looks like you did it correctly for all the other ones, so once this is resolved this is good to go!

Reply:
Thank you @brooksmtownsend I'm clueless why checks are failing can you please help me ?